apps/docs/content/docs/editor.mdx

```diff
@@ -12,11 +12,11 @@ keywords:
   - select
 ---
 
-The `Editor` class is the main control class for tldraw's editor. You can use it to manage the editor's internal state, make changes to the document, or respond to changes that have occurred.
+The `Editor` class is the main way of controlling tldraw's editor. You can use it to manage the editor's internal state, make changes to the document, or respond to changes that have occurred.
 
-By design, the `Editor`'s surface area is [very large](/gen/editor/Editor-class). While that makes it difficult to fully document here, the general rule is that everything is available via the `Editor`. Need to create some shapes? Use `editor.createShapes()`. Need to delete them? Use `editor.deleteShapes()`. Need a sorted array of every shape on the current page? Use `editor.sortedShapesArray`.
+By design, the `Editor`'s surface area is [very large](/gen/editor/Editor-class). Almost everything is available through it. Need to create some shapes? Use `editor.createShapes()`. Need to delete them? Use `editor.deleteShapes()`. Need a sorted array of every shape on the current page? Use `editor.sortedShapesArray`.
 
-Rather than document everything, this page is intended to give a broad idea of how the `Editor` class is organized and some of the architectural concepts involved.
+This page gives a broad idea of how the `Editor` class is organized and some of the architectural concepts involved. The full reference is available in the [Editor API](/gen/editor/Editor-class).
```

Comment:
I think we could speak more positively about our lovely Editor class :)

(but link to the Editor API as well, because we actually do document everything now - as of last week 🎉 )